setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import setup
 
-_PACKAGE_NAME = 'mosaicml-llm'
+_PACKAGE_NAME = 'llm-foundry'
```

Comment:
Should we be dropping the `mosaicml-` part? This is the name that will show up in `pip list` and I think our other packages are like `mosaicml-examples`, and `mosaicml-cli`, etc.

@hanlint if you have a sec curious what you prefer

Reply:
Checked with @hanlint and I think `llm-foundry` is fine.